users/tests/test_user.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from django.test import Client
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def user():
+    return User.objects.create_user('Test-user', 'Test-user@mail.com', 'Password')
+
+
+@pytest.mark.django_db
+def test_user_creation(user):
+    assert User.objects.filter(pk=user.id).exists()
+    assert User.objects.get(pk=user.id) == user
+
+
+@pytest.mark.django_db
+def test_user_delete(user):
+    assert User.objects.filter(pk=user.id).exists()
+    User.objects.get(pk=user.id).delete()
+    assert User.objects.filter(pk=user.id).exists() is False
```

Comment:
It's more acceptable to use `assert not...` instead of `is False` (though it's not a mistake)

Reply:
Changed to `assert not`